src/BaGetter.Database.MySql/BaGetter.Database.MySql.csproj

```diff
@@ -2,9 +2,10 @@
 
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
-    <LangVersion>latest</LangVersion>
+
     <PackageTags>NuGet</PackageTags>
     <Description>The libraries to host BaGetter on MySQL.</Description>
+    <PackageReadmeFile>readme.md</PackageReadmeFile>
```

Comment:
The db provider projects have no fitting readme files either. They just describe how to add a migration for each db provider.